README.md

```diff
@@ -72,20 +72,16 @@ See the [toolkit documentation](https://github.com/actions/toolkit/blob/master/R
 
 Actions are run from GitHub repos.  We will create a releases branch and only checkin production modules (core in this case). 
 
-Comment out node_modules in .gitignore and create a releases/v1 branch
-```bash
-# comment out in distribution branches
-# node_modules/
-```
-
+Create a releases/v1 branch:
 ```bash
 $ git checkout -b releases/v1
 $ git commit -a -m "prod dependencies"
 ```
 
+Then run [ncc](https://github.com/zeit/ncc) and push the results:
 ```bash
-$ npm prune --production
-$ git add node_modules
+$ npm run pack
```

Comment:
I created a project from this template today and I couldn't get the `dist` folder to update without running both build and pack: `npm run build && npm run pack`

BTW, I was also having problems with the `test` job in the workflow:

```
  test: # make sure the action works on a clean machine without building
    runs-on: ubuntu-latest
    steps:
    - uses: actions/checkout@v2
    - uses: ./
      with: 
        milliseconds: 1000
```

I was basically just trying to print `Hello World` and it wouldn't print anything.  That's because I needed to build, pack and commit the `dist` directory before it would run my new changes.  Maybe the `Code in Master` section of the README could be updated?  That's what I was following when I was making my first change.